proof/proofd/src/XrdProofdProtocol.cxx

```diff
@@ -524,7 +524,7 @@ void XrdProofdProtocol::Reset()
       fAuthProt->Delete();
       fAuthProt = 0;
    }
-   memset(&fSecEntity, 0, sizeof(fSecEntity));
+   fSecEntity = XrdSecEntity();
```

Comment:
XrdSecEntity now provides a Reset method that does what needed here.
I would rather call that instead of redifining via the constructor:

fSecEntity.Reset();

Reply:
Thanks, Gerri! I will fix the commit to use that then.